webui/react/src/pages/ExperimentDetails.tsx

```diff
@@ -29,6 +29,7 @@ const ExperimentDetails: React.FC = () => {
   const [ canceler ] = useState(new AbortController());
   const [ experiment, setExperiment ] = useState<ExperimentBase>();
   const [ trial, setTrial ] = useState<TrialDetails>();
+  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
   const [ valHistory, setValHistory ] = useState<ValidationHistory[]>([]);
```

Comment:
this is a new one for me, but you can write `const [ , setValHistory ] = useState` to not receive a variable
removes need for the eslint exception

Reply:
yeah so the only place it was being used [before](https://github.com/trentwatt/determined/blob/de46c36d7e46300cc4c9ce902f2d335f32d90bfe/webui/react/src/pages/ExperimentDetails.tsx#L46) was to see if it was equal to the previous value. 

i didn't want to just get rid of it, at least this way we call attention to its not being used when comes time to merge.